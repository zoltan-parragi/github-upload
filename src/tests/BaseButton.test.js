import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import BaseButton from '../components/Buttons';

describe('BaseButton', () => {
	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<BaseButton>base button</BaseButton>, div);
		ReactDOM.unmountComponentAtNode(div);
	});
	test('has a valid snapshot', () => {
		const component = renderer.create(
			<BaseButton>base button</BaseButton>
		);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});