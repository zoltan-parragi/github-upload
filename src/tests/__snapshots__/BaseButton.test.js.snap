// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton has a valid snapshot 1`] = `
<button
  className=""
  type="button"
>
  base button
</button>
`;
